caikit/runtime/servicers/global_predict_servicer.py

```diff
@@ -95,8 +95,10 @@ def __init__(
         inference_service: ServicePackage,
         use_abortable_threads: bool = get_config().runtime.use_abortable_threads,
     ):
+        self._started_metering = False
```

Comment:
This flag is added to make sure we don't try to `stop_metering` multiple times and invoke `self.rpc_meter.flush_metrics` multiple times, as it gives `ValueError: I/O operation on closed file.` when invoked if the metering file is already closed